suite.yml

```diff
@@ -10,58 +10,44 @@ section:
         url: https://github.com/cyberark/conjur
         description: Conjur OSS server. Conjur comes built-in with custom authenticators
           for Kubernetes, OpenShift, AWS IAM, OIDC, and more.
-        certification: "certified"
-        version: v1.4.4
+        version: v1.5.1
```

Comment:
We need to include-back our older pins first if we want a reasonably diffed 0.0.0 version first